bitnami/fluent-bit/README.md

```diff
@@ -0,0 +1,275 @@
+| `metrics.serviceMonitor.labels`                     | Extra labels for the ServiceMonitor                                                                                                                                    | `{}`                     |
+| `metrics.serviceMonitor.jobLabel`                   | The name of the label on the target service to use as the job name in Prometheus                                                                                       | `""`                     |
+| `metrics.serviceMonitor.honorLabels`                | honorLabels chooses the metric's labels on collisions with target labels                                                                                               | `false`                  |
+| `metrics.serviceMonitor.interval`                   | Interval at which metrics should be scraped.                                                                                                                           | `""`                     |
+| `metrics.serviceMonitor.scrapeTimeout`              | Timeout after which the scrape is ended                                                                                                                                | `""`                     |
+| `metrics.serviceMonitor.metricRelabelings`          | Specify additional relabeling of metrics                                                                                                                               | `[]`                     |
+| `metrics.serviceMonitor.relabelings`                | Specify general relabeling                                                                                                                                             | `[]`                     |
+| `metrics.serviceMonitor.selector`                   | Prometheus instance selector labels                                                                                                                                    | `{}`                     |
+
+## Configuration and installation details
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+### Persistence
+
+The [Bitnami Fluent Bit](https://github.com/bitnami/containers/tree/main/bitnami/fluent-bit) image stores the trace onto an external database. Persistent Volume Claims are used to keep the data across deployments.
```

Comment:
Which database? I do not see any subchart nor reference to a database.

Reply:
Re-written[fc961c7](https://github.com/bitnami/charts/pull/15453/commits/fc961c776cb00e6f98e5f39b1ecad58429705cf1)